frontend/src/components/Field/Rating/RatingOption.tsx

```diff
@@ -202,6 +205,7 @@ export const RatingOption = forwardRef<RatingOptionProps, 'input'>(
         <input
           type="radio"
           aria-checked={selectedValue === value}
+          aria-label={`${value} ${variant}${value === 1 ? '' : 's'}`}
```

Comment:
/nitNothingToDo: Just a note that when (IF!) we ever introduce language localisation, pluralisation should not be done by concatenation.

Reply:
ndb but i wonder if we can use simplur for this? 
```javascript
aria-label={simplur`${value} ${variant}${[value]}[|s]`}

Reply:
ndb or nbd? 😜🤔